sklearn/linear_model/least_angle.py

```diff
@@ -195,7 +195,6 @@ def lars_path(X, y, Xy=None, Gram=None, max_iter=500,
             sys.stdout.write('.')
             sys.stdout.flush()
 
-    tiny = np.finfo(np.float).tiny  # to avoid division by 0 warning
     tiny32 = np.finfo(np.float32).tiny  # to avoid division by 0 warning
```

Comment:
for the record tiny32 was introduced here https://github.com/scikit-learn/scikit-learn/commit/294d4b605728cd3ed40dbfdf508a8517cb7695f8